src/Infrastructure/Service/TaskOptionBuilder.php

```diff
@@ -104,10 +105,18 @@ protected function addPlaceholderOptions(TaskInterface $task, array $options): a
                 continue;
             }
 
+            $mode = $placeholder->isOptional() ? InputOption::VALUE_OPTIONAL : InputOption::VALUE_REQUIRED;
```

Comment:
Move mode building in separate method pls.

Reply:
Fixed